src/golang/lib/models/views/id.go

```diff
@@ -0,0 +1,8 @@
+package views
+
+import "github.com/google/uuid"
+
+// ObjectID is a wrapper around any ID
+type ObjectID struct {
+	ID uuid.UUID `db:"id" json:"id"`
```

Comment:
This can go to future refactors. But I think in these situations we should simply return id / list of ids from the query. If a struct is required to fetch the return value it should be an private view within that repo, rather than a public one